docs-ref-services/key-vault.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Azure Key Vault libraries for Python
+description: Reference documentation for the Python client libraries for Azure Key Vault
+author: lisawong19
+keywords: Azure, Python, SDK, API, Keys, Key Vault, Authentication, Secret, key, security
+manager: douge
+ms.author: liwong
+ms.date: 06/26/2017
+ms.topic: article
+ms.devlang: python
+ms.service: keyvault
+---
+
+# Overview
+
+Create, update, and delete keys and secrets in Azure Key Vault with the client libraries.
+
+Use the Azure Key Vault management libraries to create key vaults, authorize applications, and manage permissions. 
+
+Learn more about [Azure Key Vault](/azure/key-vault/key-vault-whatis).
+
+## Install the libraries
+
+### Client library
+
+```bash
+pip install azure-keyvault
+```
+
+### Management 
+
+```bash
+pip install azure-mgmt-keyvault
+```
+
+## Example
+
+Retrieve a [JSON web key](https://tools.ietf.org/html/draft-ietf-jose-json-web-key-18) from a Key Vault.
+
+```python
+client = KeyVaultClient(credentials)
```

Comment:
Include the `import` statement to show which of the two libraries is required, and where the referenced classes/methods are imported from.